src/@types/Asset.ts

```diff
@@ -34,7 +34,7 @@ export interface AssetNft {
    * 4	Ordering is temporary disabled.
    * @type {number}
    */
-  state: 0 | 1 | 2 | 3 | 4
+  state: 0 | 1 | 2 | 3 | 4 | 5
```

Comment:
can you pls include this state in the jsdoc also, to have a short desc about this new state ?